README.md

```diff
@@ -1,320 +1,66 @@
-```sh
-celestia-appd tx staking create-validator \
- --amount=1000000000celes \
- --pubkey=$(celestia-appd tendermint show-validator) \
- --moniker=space \
- --chain-id=devnet-2 \
- --commission-rate=0.1 \
- --commission-max-rate=0.2 \
- --commission-max-change-rate=0.01 \
- --min-self-delegation=1000000000 \
- --from=$node_name \
- --keyring-backend=test
-```
-
-which should prompt for:
-
-```sh
-confirm transaction before signing and broadcasting [y/N]:
-```
-
-Inputting `y` should provide an output similar to:
-
-```sh
-code: 0
-codespace: ""
-data: ""
-gas_used: "0"
-gas_wanted: "0"
-height: "0"
-info: ""
-logs: []
-raw_log: '[]'
-timestamp: ""
-tx: null
-txhash: <tx-hash>
-```
-
-You should now be able to see your validator from a block explorer such as http://celestia.observer:3080/validators.
-
-### Troubleshooting Validator Node setup
-
-If you get an error such as 
-
-```Error: <keyname>: key not found```,
-
-this means your key, the field referenced by the `--from` option, does not exist.
-
-You can fix this by adding your key manually to the keyring via:
-
-```
-celestia-appd keys add --recover <keyname>
-```
-
-followed by a prompt to enter a bip39 mnemonic, which is the mnemonic that was created as part of `1_create_key.sh` script in the first step.
-
-You'll also be asked for a passphrase which is an input you have to define.
-
-After this fix repeat step 6.
+- [Run Celestia Devnet](#run-celestia-devnet)
+  - [Celestia 101(Overview)](#celestia-101overview)
+    - [Celestia Full Node](#celestia-full-node)
+    - [Celestia Light Client](#celestia-light-client)
+  - [Installation guide](#installation-guide)
+  - [Troubleshoot](#troubleshoot)
 
+## Celestia 101(Overview)
+<i>Conceptually</i>, we have 2 main components in the network: 
+1. `Celestia Full Node`.
+2. `Celestia Light Client`.
 
-## Running a Celestia Node
-### Pre-Requisites
-You need to have the trusted hash in order to initialize the full celestia node
-In order to know the hash, you need to query the celestia-app:
-```sh
-curl -s http://localhost:26657/block?height=1 | grep -A1 block_id | grep hash
-```
+> Note: When you see formated text of those 2 above(`Celestia Full Node`, `Celestia Light Client`), remember that we are refering to a <i>conceptual</i> view. Standard formated text is a technical view 
 
-### Full Node Configuration
-1. Initialize the full node
-```sh
-celestia full init --core.remote <ip:port of celestia-app> --headers.trusted-hash <hash_from_celestia_app>
-```
+### Celestia Full Node
+When we are mentioning `Celestia Full Node`, we are relating to a combination of 2 components: 
 
-Example:
-```sh 
-celestia full init --core.remote tcp://127.0.0.1:26657 --headers.trusted-hash 3BDFBD7E2D97215CFA600DD8B39AAEECC015E43FEE7B8A4D8A8B630E8B4D4006
-```
+- Celestia Application (lives in [celestia-app](https://github.com/celestiaorg/celestia-app) repo)
+- Celestia Full Node (lives in [celestia-node](https://github.com/celestiaorg/celestia-node) repo)
```

Comment:
I believe we're referring to the "Celestia Full Nodes" as "Bridge Nodes" now

Reply:
For the devnet, we have Celestia Full Nodes and Celestia Light Nodes (clients) 
Let's not confuse ppl with what we have decided with post-devnet naming

In the testnet doc, we should definitely rework this section to the updated schema